src/integration-test/java/com/commercetools/project/sync/InventorySyncMultiChannelIT.java

```diff
@@ -0,0 +1,206 @@
+package com.commercetools.project.sync;
+
+import static com.commercetools.project.sync.util.IntegrationTestUtils.cleanUpProjects;
+import static com.commercetools.project.sync.util.IntegrationTestUtils.createITSyncerFactory;
+import static com.commercetools.project.sync.util.SphereClientUtils.CTP_SOURCE_CLIENT;
+import static com.commercetools.project.sync.util.SphereClientUtils.CTP_TARGET_CLIENT;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.commercetools.project.sync.inventoryentry.InventoryEntrySyncer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.channels.ChannelDraft;
+import io.sphere.sdk.channels.ChannelRole;
+import io.sphere.sdk.channels.commands.ChannelCreateCommand;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+import io.sphere.sdk.inventory.InventoryEntryDraftBuilder;
+import io.sphere.sdk.inventory.commands.InventoryEntryCreateCommand;
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.models.Reference;
+import io.sphere.sdk.types.CustomFieldsDraft;
+import io.sphere.sdk.types.FieldDefinition;
+import io.sphere.sdk.types.StringFieldType;
+import io.sphere.sdk.types.TypeDraft;
+import io.sphere.sdk.types.TypeDraftBuilder;
+import io.sphere.sdk.types.commands.TypeCreateCommand;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.IntStream;
+import javax.annotation.Nonnull;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import uk.org.lidalia.slf4jext.Level;
+import uk.org.lidalia.slf4jtest.TestLogger;
+import uk.org.lidalia.slf4jtest.TestLoggerFactory;
+
+@Disabled(
+    "Disabled for normal builds, "
+        + "This scenario created to evaluate multiple channel use case"
+        + "see: https://github.com/commercetools/commercetools-project-sync/issues/301")
+public class InventorySyncMultiChannelIT {
+  private static final TestLogger testLogger =
+      TestLoggerFactory.getTestLogger(InventoryEntrySyncer.class);
+
+  public static final String SKU_1 = "SKU_ONE";
+  public static final String SKU_2 = "SKU_TWO";
+  public static final String SUPPLY_CHANNEL_KEY_1 = "channel-key_1";
+  public static final String SUPPLY_CHANNEL_KEY_2 = "channel-key_2";
+
+  public static final String CUSTOM_TYPE = "inventory-custom-type-name";
+  public static final String CUSTOM_FIELD_NAME = "inventory-custom-field-1";
+
+  @BeforeEach
+  void setup() {
+    testLogger.clearAll();
+    cleanUpProjects(CTP_SOURCE_CLIENT, CTP_TARGET_CLIENT);
+    create249InventoryEntry(CTP_SOURCE_CLIENT);
+    create249InventoryEntry(CTP_TARGET_CLIENT);
```

Comment:
I would put this info "setupProjectData"

Reply:
You're right, but for now let's keep it, as the source is already disabled. I will keep the test in case a follow up ticket for the other potential issues of InventoryEntry sync.